spec/pdfs/response_report_spec.rb

```diff
@@ -33,7 +33,8 @@ def complete_response
       expect(page_1).to include('In de onderstaande tabel ziet u de resultaten van de ingevulde vragenlijst.')
     end
 
-    it 'contains the date of the response' do
+    it 'contains the date of the request and response' do
+      expect(page_2).to include('Aangevraagd op')
```

Comment:
Op zich zitten daar al basic tests voor in Rails/i18n. Wij hebben geen dynamische timezones voor gebruikers en gebruiken ook geen queries met datum/tijd vergelijkingen, dus volgens mij heeft het weinig zin om dit heel erg uitgebreid te gaan testen.